### ISSUE TITLE
Sort By Column Error

### ISSUE THREAD

**Issue by GilbertQue** [TabularEditor/TabularEditor3#625]
Hi there

I am using the latest version of Tabular Editor 3 (3.3.4)

And now when I select a column for the Sort By Column I am getting the following error as shown below
![image](https://user-images.githubusercontent.com/72849240/184044536-33f94f52-1c69-4ee1-82cf-d048b01ce95e.png)

Thanks for all the awesome work!

**Comment by otykier** [comment]
Yeah, we noticed this shortly after release :-( 

Type the name of the column in the property grid, or use a c# script as a workaround for now:

```csharp
Selected.Column.SortByColumn = SelectColumn(Selected.Column.Table.Columns);
```

**Comment by otykier** [comment]
Here's a slightly better version of the script which doesn't clear the SortBy column when hitting "Cancel":

```csharp
var newSortByColumn = SelectColumn(Selected.Table, Selected.Column.SortByColumn);
if(newSortByColumn != null)
    Selected.Column.SortByColumn = newSortByColumn;
```

**Comment by GilbertQue** [comment]
Awesome thanks I will try that.

**Comment by otykier** [comment]
Fixed in [3.3.5](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.3.5). Thanks for reporting!
